reference/docs-conceptual/Crescendo/get-started/choose-command-line-tool.md

```diff
@@ -0,0 +1,73 @@
+---
+description: How to select a command-line tool to amplify using Crescendo.
+ms.date: 03/08/2022
+title: Choosing a command-line tool to amplify
+---
+# Choosing the command-line tool for Crescendo
+
+Developing a cmdlet with Crescendo is a rapid, reliable way to _amplify_ a command-line tool to
+produce a cmdlet-like experience. Many times, the tool you need to work with is directly tied to the
+technology you are attempting to automate. This makes the tool choice simple. However, not all tools
+work the same way. To optimize your investment when developing a cmdlet, consider the following
+criteria:
+
+## Command-line tools that make good candidates
+
+- **The original tool is difficult to use.**
+
+  If the tool is simple to use and provides the information you need, there is no need to create a
+  cmdlet. However, this isn't always the case with all command-line tools. Many command-line tools
+  have their own unique syntax, parameters, and output that make it difficult for inexperienced
+  administrators to use the command. Converting the command into a cmdlet provides all the benefits
+  of cmdlet discovery, syntax consistency, and structured output as objects.
+
+- **The command-line tool output is difficult to use in automation.**
+
+  Command-line tools output their information to the screen as string data. This is not the
+  structured data (objects) that PowerShell expects in the pipeline, which prevents you from using
+  PowerShell cmdlets, such as `Where-Object` and `ForEach-Object`. Crescendo assists you with
+  _amplifying_ the command-line tool experience to a cmdlet so that the tool can participate in the
```

Comment:
I'm not sure on the phrasing here, something about the rhythm felt off when I said it out loud. Do we want to highlight that Crescendo is providing a cmdlet that amplifies the command-line tool, that Crescendo's amplification _is_ a cmdlet, that Crescendo is specifically amplifying a command-line tool for pipeline functionality, or something else?

```suggestion
  _amplifying_ the command-line tool experience so that the tool can participate in the
```